asv_bench/benchmarks/io_bench.py

```diff
@@ -10,16 +10,21 @@
 
 class frame_to_csv(object):
     goal_time = 0.2
+    fname = '__test__.csv'
 
     def setup(self):
         self.df = DataFrame(np.random.randn(3000, 30))
 
     def time_frame_to_csv(self):
-        self.df.to_csv('__test__.csv')
+        self.df.to_csv(self.fname)
+
+    def teardown(self):
+        os.remove(self.fname)
```

Comment:
This can raise an error if `os.remove` function fails.  I would suggest that you try to catch any errors here in a `try-except` (some of the other files that you modified do this).

Reply:
I would try except all of these teardowns, maybe create a generic function at the top of this file to do so.

Reply:
Or maybe I should create one base class which only implements teardown method and all bench classes would inherit after it. @gfyoung, @jreback - do you consider this a good idea? Or generic function at the top would be better?

Reply:
yep I like that better actually. I *think* all of the classes that need it are in this module anyhow.